File: Triangles/LeftAngleTriangle/hollowLeftTriangleWith4Pyramids/HollowLeftTriangleWith4Pyramids.java
public class HollowLeftTriangleWith4Pyramids {
        public static void main(String args[]) {
        int numRows = 9;
        int colVal;
        int rowVal;
        int temp = 2;
        char patternSymbol = '*';

        for(rowVal = 1; rowVal <= numRows; rowVal++) {
            for(colVal = 1; colVal <= rowVal; colVal++) {
                if(colVal == 1 || colVal == rowVal || rowVal == numRows) {
                    System.out.print(patternSymbol + " ");
                } else if(colVal == (numRows/2) + 1 || rowVal == (numRows / 2) + 1) {
                    System.out.print(patternSymbol + " ");
                } else if(rowVal == (numRows/2) + temp && colVal == temp) {
                    System.out.print(patternSymbol + " ");
                    temp++;
                } else {
                    System.out.print("  ");
                }
            }
            System.out.println(""); 
        }
    }
}

File: Triangles/LeftAngleTriangle/hollowLeftTriangleWith4Pyramids/HollowLeftTriangleWith4Pyramids.py
numRows = 9
temp = 2
patternSymbol = '*'

for rowVal in range(1, numRows + 1):
    for colVal in range(1, rowVal + 1):
        if colVal == 1 or colVal == rowVal or rowVal == numRows:
            print(patternSymbol, "", end="") # heer, I am only using this "" because, sep parameter is already giving us one space as sep=" "
        elif colVal == (numRows//2) + 1 or rowVal == (numRows // 2) + 1:
            print(patternSymbol, "", end="")
        elif rowVal == (numRows//2) + temp and colVal == temp:
            print(patternSymbol, "", end="")
            temp += 1
        else:
            print("  ", end="")
    print()   # for new line